### PR TITLE
Add Project toml

### PR DESCRIPTION
**Pull request by Ayushk4**
The version in the project. toml has been mentioned the same as last release (0.2.0)

Also refer #24

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+[deps]
+LightXML = "9c8b4983-aa76-5018-a973-4c85ecc9e179"
+DataDeps = "124859b0-ceae-595e-8997-d05f6a7a8dfe"
+Glob = "c27321d9-0574-5035-807b-f59d2c89b15c"
+InternedStrings = "7d512f48-7fb1-5a58-b986-67e6dc259f01"
+StringEncodings = "69024149-9ee7-55f6-a4c4-859efe599b68"
+WordTokenizers = "796a5d58-b03d-544a-977e-18100b691f6e"
+MultiResolutionIterators = "396aa475-d5af-5b65-8c11-5c82e21b2380"
+CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
+
```
Comment (oxinabox) [inline Project.toml]: Needs a `[compat]` section that at least contains a `julia="1"`

File: Project.toml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name = "CorpusLoaders"
+uuid = "214a0ac2-f95b-54f7-a80b-442ed9c2c9e8"
+license = "MIT"
+desc = "A variety of loaders for various NLP corpora."
+version = "0.2.0"
```
Comment (oxinabox) [inline Project.toml]: Let's just bump to 0.3.

**Comment by Ayushk4** [comment]
Appveyor seems to be failing since the Wiki corpus takes too long to download. Should I edit appveyor.yml to allow failure on it, just like it what earlier done for Travis?

**Comment by Ayushk4** [comment]
WikiCorpus Tests disabled on Appveyor.

**Pull request merged**